source/includes/fundamentals/code-snippets/CRUD/bulkOps.go

```diff
@@ -52,13 +53,15 @@ func main() {
 
 	fmt.Println("\nInsertOneModel:\n")
 	{
+		// Creates write models that specify insert operations
 		// begin bulk insert model
 		models := []mongo.WriteModel{
 			mongo.NewInsertOneModel().SetDocument(Book{Title: "Beloved", Author: "Toni Morrison", Length: 324}),
 			mongo.NewInsertOneModel().SetDocument(Book{Title: "Outline", Author: "Rachel Cusk", Length: 258}),
 		}
 		// end bulk insert model
 
+		// Performs the bulk write and prints the number of inserted documents
```

Comment:
S: just based on the format of the Java bulk write comments
```suggestion
		// Runs the bulk write operation and prints the number of inserted documents
```